supabase/migrations/rollback_20260704140000_donor_matching_merge_candidates.sql

```
@@ -0,0 +1,18 @@
+-- Rollback for 20260704140000_donor_matching_merge_candidates.sql
+-- Drops the new tables and the donors redirect columns. Idempotent (IF EXISTS).
+-- Safe ordering: audit/candidate tables first (they FK donors), then donors columns.
+
+DROP TABLE IF EXISTS public.donor_merge_audit;
+DROP TABLE IF EXISTS public.donor_merge_candidates;
+
+ALTER TABLE public.donors
+  DROP CONSTRAINT IF EXISTS donors_merged_into_same_tenant_fk,
+  DROP CONSTRAINT IF EXISTS donors_merged_requires_tenant_check;
+
+DROP INDEX IF EXISTS public.donors_tenant_id_id_uidx;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)